packages/client/README.md

```diff
@@ -51,6 +51,8 @@ client you can use to finish the pre-authorized code flows.
 This initiates the client using a URI obtained from the Issuer using a link (URL) or QR code typically. We are also
 already fetching the Server Metadata
 
+
+Using openid-initiate-issuance scheme
```

Comment:
As mentioned that is ancient. It is called credential offer for quite some time. So if you add documentation, don't use old terms

Reply:
It was here already. Removed